common/views/components/ErrorPage/ErrorPage.tsx

```diff
@@ -60,32 +60,54 @@ const TogglesMessage: FunctionComponent = () => {
   const [toggles, setToggles] = useState<string[]>([]);
 
   useEffect(() => {
-    setToggles(
+    setToggles(() => {
       // dangerouslyGetEnabledToggles returns a list with of all toggle cookies that are set.
       // Those prefixed with a ! have a false value and we only need to show the toggles with a value of true here
-      dangerouslyGetEnabledToggles(getCookies()).filter(v => !v.startsWith('!'))
-    );
+      const activeTogglesInBrowser = dangerouslyGetEnabledToggles(
+        getCookies()
+      ).filter(v => !v.startsWith('!'));
+
+      // Get the readable name
+      if (activeTogglesInBrowser.length > 0) {
+        const allToggles = [...togglesList.toggles, ...togglesList.tests];
+        const activeToggleNames = activeTogglesInBrowser
+          .map(
+            id =>
+              Object.values(allToggles).find(toggle => toggle.id === id)?.title
+          )
+          .filter(f => f);
+        return activeToggleNames as string[];
+      } else {
+        return [];
+      }
+    });
   }, []);
 
   return toggles.length > 0 ? (
-    <Layout gridSizes={gridSize8()}>
-      <MessageBar>
-        <Space $h={{ size: 's', properties: ['margin-right'] }}>
-          <Icon icon={underConstruction} />
-        </Space>
-        <Space $h={{ size: 's', properties: ['margin-right'] }}>
-          You have the following{' '}
-          <a href="https://dash.wellcomecollection.org/toggles">toggles</a>{' '}
-          enabled:{' '}
-          {toggles.map((t, i) => (
-            <Fragment key={t}>
-              <strong>{t}</strong>
-              {i !== toggles.length - 1 && <>, </>}
-            </Fragment>
-          ))}
-        </Space>
-      </MessageBar>
-    </Layout>
+    <Space $v={{ size: 'l', properties: ['margin-top'] }}>
+      <Layout gridSizes={gridSize8()}>
+        <MessageBar>
+          <Space $h={{ size: 's', properties: ['margin-right'] }}>
+            <Icon icon={underConstruction} />
+          </Space>
+          <Space $h={{ size: 's', properties: ['margin-right'] }}>
+            You have the following toggles enabled:
+            <ul>
+              {toggles.map(t => (
+                <li key={t}>
+                  <strong>{t}</strong>
+                </li>
+              ))}
+            </ul>
+            This could be what is causing this page to error,{' '}
+            <a href="https://dash.wellcomecollection.org/toggles">
+              please try resetting your toggles
```

Comment:
Absolutely a non-issue, but do we think users know they then have to select the reset button on the page once they've gone to the toggles dash? Not blocking!

Reply:
That's a good question! I don't know how familiar people are with it, but as the button is quite high up and red and the messaging is similar, I'm tempted to trust they'll find it intuitive.
I also think that using the title of the toggle in the message will also remind them of that time where they had to turn it "on" and see they could turn it "off" (which would have the same effect as resetting). Using the id was probably more useful to us than to them and would not have meant much. 
What do you think?